action.yml

```diff
@@ -24,5 +24,5 @@ runs:
         curl -X POST \
           -H "Authorization: Bearer ${{ inputs.token }}" \
           -H "Content-Type: application/hcl" \
-          -d "@$${{ inputs.job }}" \
-          "${{ inputs.deployment_url }}"
+          -d "@${{ inputs.job }}" \
+          "${{ inputs.endpoint}}"
```

Comment:
@gamechanger1s - the README must be updated to reflect the new variable. My suggestion would be to keep this the same as before. If you must change this, do that as a separate PR.